README.md

```diff
@@ -102,6 +102,25 @@ the secrets stored in temp files and in the Python process environment are gone.
 
     This flag can be useful for when you have secrets that you don't need access to for development. For example API keys for monitoring tools. This flag can be used multiple times.
 
+* `-e, --environment` Specify section (environment) to parse from secret YAML
+
+    This flag specifies which specific environment/section to parse from the secrets YAML file (or string). In addition, it will also enable the usage of a `common` section which will be inherited by other sections/environments. In other words, if your `secrets.yaml` looks something like this:
+
+```yaml
+common:
```

Comment:
I agree that adding environment-specific config in the app code repo is a bit unclean, as it introduces specific configuration in the application code -- something summon tries to help avoid, by abstracting it and instead declaring dependencies, so to speak. That said, I can see this can be useful in some cases; I'm just a bit worried this might actually encourage it. Perhaps the `-D` option should be advertised more?